tools/clang/test/CodeGenSPIRV/spirv.debug.opline.precedence.hlsl

```diff
@@ -0,0 +1,25 @@
+// Run: %dxc -T cs_6_0 -E main -Zi
```

Comment:
What is the purpose of this test? This test was not added to `CodeGenSpirvTest.cpp`